---
title: "BIOL 210 Problem Set 4 KEY"
date: "Week 4, Sep 23-27"
output: 
  html_document:
    toc: TRUE
    toc_depth: 3
  pdf_document:
    toc: TRUE
    toc_depth: 3
pkgdown:
  as_is: true
---
<text style="color:blue">This is the key for PS 4. Before reading this document, you should have completed the problems. Use this key to check and correct your work BEFORE submitting the corrected version via the google form.</text>

<text style="color:blue">You should compare your responses with this key and make any changes in another font color. Be sure to explain why you got things wrong (showing that now you understand) as well as providing corrected responses.</text>


# Question Key

<ol>
<li>Let’s work on thinking carefully about genes and the processes in which they are involved.</li>

<ol type="a">
<li>What is gene expression? How does gene expression differ from DNA replication?</li>

<text style="color:blue">Gene expression is the process by which the information encoded in DNA is used to create RNA and proteins. Thus, gene expression is how cells perform actions, using their library of information in DNA to produce outputs (proteins, RNA) that can carry out specific activities. This is very different from DNA replication, which merely makes a copy of the DNA but does not use the DNA to do anything other than make more DNA. </text>

<ul>
<li><text style="color:blue">Gene expression produces RNA and protein. DNA replication produces more DNA (a copy).</text></li>

<li><text style="color:blue">DNA replication only* happens in preparation for cell reproduction, once in the life of a cell. DNA replication is happening all the time, as part of the regular life of a cell maintaining its function.</text></li>

<li><text style="color:blue">DNA replication occurs in the nucleus in eukaryotes. In eukaryotes, gene expression begins in the nucleus but part of it happens in the cytosol.</text></li>
</ul>

<text style="color:blue">*Note: there are situations in which a cell will make many copies of the DNA for other reasons, sometimes unknown. In plants, this happens a lot in somatic tissues (like leaves) and is known as endoreduplication.</text>

<li>Describe the basic structure of a eukaryotic gene. What do you think are the “parts” of a gene? What important features that we should take note of?</li>

<text style="color:blue">The "parts" of a gene include:</text>

<ul>
<li><text style="color:blue">>Exons = the part of the gene’s DNA sequence that encodes amino acids. Most genes have more than one exon sequence. The exons are the “coding region” of the gene.</text</li>

<li><text style="color:blue">Introns = this part of the gene’s DNA does NOT encode amino acids and is usually excised from the RNA before protein is made. This is considered “non-coding” DNA even though sometimes it contains sequences that act as signals of where to splice the RNA. Most genes have more than one intron sequence.</text></li>

<li><text style="color:blue">Promoter = this is a relatively short sequence of DNA (100-1000 base pairs) that is located just before the encoding sequence of the gene. Serves as a binding site for the RNA polymerase, thus enabling the polymerase to begin transcription of the gene.</text></li>

<li><text style="color:blue">Terminator = sequence that signals the end of the coding region of the gene, where transcription stops. Reaching this sequence leads to the newly created RNA transcript being released from the RNA polymerase.</text></li>
</ul><br>

<text style="color:blue">Not strictly part of the coding region of the gene but important to transcription of the gene:</text>

<ul>
<li><text style="color:blue">Regulatory sequences = regions of DNA sequence that serve as binding sites for proteins that can change the probability or rate of transcription (for example, by interacting with the transcription complex). These are usually outside of the gene itself, often upstream (before) the gene sequence. They may be near or far from the gene (up to 50,000 base pairs away).</text></li>
<ul>
<li><text style="color:blue">Enhancer = sequence that is recognized and bound by an activator protein (a transcription factor) that increases transcription of the gene.</text></li>

<li><text style="color:blue">Silencer (operator) = sequence recognized and bound by an active repressor protein (a transcription factor) that reduces or prevents transcription of the gene.</text></li>
</ul></ul><br>

<li>Explain the slit-shaped eye phenotype found in female fruit flies that have two copies of the Bar allele (<i>X<sup>B</sup>X<sup>B</sup></i>) for the BarH1 gene. Why do you think the Bar allele (<i>X<sup>B</sup></i>) leads to a misshapen adult eye?

Resources to read about the BarH1 gene, if you don’t remember from videos:

https://www.ndsu.edu/pubweb/~mcclean/plsc431/chromstruct/chrmo2.htm

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5105861/

https://www.sdbonline.org/sites/fly/gene/barh1-4.htm

<text style="color:blue">Here we need to think about how the Bar allele (<i>X<sup>B</sup></i>) is different from the wild-type allele (<i>X<sup>+</sup></i>). The Bar allele has two copies of gene right next to each other on the chromosome – it’s a duplication. That means that while a wild-type individual (<i>X<sup>+</sup></i><i>X<sup>+</sup></i>) has 2 copies of the BarH1 gene, the homozygous Bar individual (<i>X<sup>B</sup></i><i>X<sup>B</sup></i>) has 4 copies of the gene. When we think about gene expression, what could it mean to have 4 copies instead of 2 copies of a gene? If all copies respond to the signal to transcribe the gene, that might mean that the individual with 4 copies make a lot more RNA for this gene and that might mean it makes a lot more of the protein that is encoded (the BarH1 gene is over-expressed). Because BarH1 encodes a transcription factor, making more of this protein is likely to change (increase or decrease) how much RNA is produced for the genes that BarH1 is regulating. Similarly, if we think about the heterozygote (<i>X<sup>+</sup></i><i>X<sup>B</sup></i>), we note that the eye is less misshapen and this individual has 3 copies, only one more than the wild-type. So, the heterozygote probably makes less BarH1 protein product than the Bar homozygote and thus has less several malformation of the eye.</text>

<text style="color:blue">Thus, we might conclude that producing a different AMOUNT of the BarH1 protein product is why the Bar allele causes a different eye shape. It’s not a difference in the shape of the protein produced nor in the function it carries out. It’s a change in how much protein is present that changes the eye phenotype.</text>
</ol>
<br>

<li>Describe the similarities and the differences in gene structure and gene expression (or regulation thereof) between prokaryotes and eukaryotes. Remember that eukaryotes includes plants, fungi, and animals so make sure your response is general and not animal-specific.</li>

<text style="color:blue">Note: I am NOT asking you to talk about specific proteins here. Notice the level of detail that I’m aiming for. If we get too specific, we lose the ability to apply the information to a broad array of organisms and situations.</text>

<text style="color:blue">Generally, gene structure is pretty similar in prokaryotes and eukaryotes. Both have exons, introns, promoters, and terminators as well as enhancer and silencer sequences. Note: prokaryotic introns are self-splicing (auto-splicing), that is, they cut themselves out of the RNA transcript.</text>

<text style="color:blue">While basic gene structure is quite similar, gene expression differs substantially between these two groups. In eukaryotes, transcription of RNA occurs in the nucleus while RNA molecules must be moved to the cytosol for translation into polypeptides (proteins). This process in eukaryotes also involves a complicated sets of steps to “process” the RNA before it leaves the nucleus: splicing out some introns and exons; adding a 5’ cap and poly-A tail to the RNA molecule (these are signals important for export from the nucleus and proper localization in the cytosol (cytoplasm). This complicated process of gene expression may give eukaryotes a broader range of proteins they can produce, giving more tailored responses to stimuli (more combinations of things are possible).</text>

<text style="color:blue">Prokaryotes, in comparison, have a more streamlined procedure. Both transcription and translation take place in the cytosol (there’s no nucleus) and there is basically no RNA processing (introns are self-splicing). In fact, ribosomes bind to RNA and begin translation even before the RNA is fully transcribed! This means that prokaryotic gene expression is much faster than in eukaryotes, possibly giving them an advantage in responding to environmental stimuli.</text>

<li>The human genome contains an estimated 25,000 genes and our bodies include about 200 different types of cells. On average, a cell in the human body typically expresses about 50% (∼12,500) of the genes in the genome. However, about 8,000 of those genes are expressed in all cell or tissue types ([Ramsköld et al. 2009](https://pubmed.ncbi.nlm.nih.gov/20011106/)). That means that about 2/3 of the genes expressed in a cell are the same genes expressed in <b>all</b> types of cells, leaving only 1/3 to distinguish different cell types from each other. Given this information, consider the following.</li>

<ol type="a">
<li>Given that we start out as a single cell, how do different cells come to have different functions? Describe how you think cells regulate gene expression to lead to cell differentiation. Think of describing what makes a heart cell look and act differently from a skin cell even though they have the same genome.</li>

<text style="color:blue">Here, like in an earlier question, remember we want to think of the overall structure of the system, not the details of specific proteins. Based on the information provided, it seems like a cell’s identity as a heart cell depends on around 4,000 genes that are being expressed differently by the heart cell in comparison to the skin cell. The other about 8,000 genes being expressed by both heart and skin cells might be encoding proteins (or RNAs) that are important for functions that all cells need to carry out, like making ribosomes. So, even though all the cells have the same DNA, they use different subsets of the genome to make RNAs and proteins that give the cells their identity.</text>

<li>Once a cell has acquired a fate (such as “I’m a skin cell”), what prevents the cell from changing its fate (say, to become a liver cell instead)? How is cell identity maintained through regulation of gene expression? </li>

<text style="color:blue">If which genes you express is important to maintaining the identity of a cell, then there must be something that prevents some genes from being expressed and makes sure that others ARE expressed. Such a process is a method of gene regulation. From the videos and reading in epigenetics, we might guess that producing the appropriate repressors and activators would allow a cell to control which genes are being expressed to make RNA or protein. In addition, some proteins (like methylases or acetylases) can add methyl groups or acetyl groups to DNA itself or to histones and these molecules act like repressors and activators, reducing or increasing transcription of a gene (but the methyl and acetyl groups are not themselves proteins).</text>
</ol>
<br>

<li>In chapter 3 of Richard Francis' <u>Epigenetics</u>, he describes a dramatic example of social interactions influencing the phenotype of male African cichlids. Some animal species take this even further. For example, in some fish species, such as the blue-headed wrasse (<i>Thalassoma bifasciatum</i>), the biological sex an immature individual develops will depend on the other fish it encounters. A juvenile wrasse that finds a coral reef not defended by an adult male will develop into a male, with sperm-producing gonads. But a juvenile wrasse on a male-defended coral reef will develop into a female, with egg-producing gonads. In some taxa, an individual may even change biological sex (i.e., gamete production) multiple times in its lifetime. Examples such as the wrasse abound among animals, as unfamiliar as they may be to us. While we sometimes think of biological sex being genetically pre-determined, these examples make clear that it is not so simple or deterministic. 

How could we alter our understanding of biological “sex determination” in order to better represent what we see in the world around us? That is, can you describe a possible mechanism or mechanisms by which sex determination (i.e., type of gametes produced) occurs in eukaryotes?
(More on variation in sex determination next week!)</li>

<text style="color:blue">There are a wide variety of reasonable responses here and we’re going to discuss how organisms do this next week both in video lectures and via case study. What I’d suggest here is to try to think clearly about what we mean (as biologists) when we say “sex” and “sex determination”. It’s clear that not all organisms use a genetic mechanism of sex determination. But at the same time, gamete production and secondary sex characteristics are clearly things an organism’s body produces based on information available in its genome – there is a genetic basis. What’s important is which gene pathways are activated or repressed in the organism’s genome. If everyone has the ability to produce either male or female gametes (or even both at once), then it’s gene regulation that determines which thing a given individual does, in response to some signal, internal or external. In organisms with sex chromosomes, it’s not so different. Genes on the X and Y chromosomes are regulatory – they turn on or off pathways of other genes (mostly on autosomes) that lead to the development of gonads and secondary sex characteristics.</text>
</ol>