<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="description" content="biol210">
<title>BIOL 210 Problem Set 7 KEY • biol210</title>
<script src="../deps/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="../deps/bootstrap-5.3.1/bootstrap.min.css" rel="stylesheet">
<script src="../deps/bootstrap-5.3.1/bootstrap.bundle.min.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- bootstrap-toc --><script src="https://cdn.jsdelivr.net/gh/afeld/bootstrap-toc@v1.0.1/dist/bootstrap-toc.min.js" integrity="sha256-4veVQbu7//Lk5TSmc7YV48MxtMy98e26cf5MrgZYnwo=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.11/clipboard.min.js" integrity="sha512-7O5pXpc0oCRrxk8RUfDYFgn0nO1t+jLuIOQdOMRp4APB7uZ4vSjspzp5y6YDtDs4VzUSTbWzBFZ/LKJhnyFOKw==" crossorigin="anonymous" referrerpolicy="no-referrer"></script><!-- search --><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- pkgdown --><script src="../pkgdown.js"></script><meta property="og:title" content="BIOL 210 Problem Set 7 KEY">
<meta property="og:description" content="biol210">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body>
    <a href="#main" class="visually-hidden-focusable">Skip to contents</a>
    

    <nav class="navbar fixed-top navbar-light navbar-expand-lg bg-light" data-bs-theme="light"><div class="container">
    
    <a class="navbar-brand me-2" href="../index.html">biol210</a>

    <small class="nav-text text-muted me-auto" data-bs-toggle="tooltip" data-bs-placement="bottom" title="">1.24.2</small>

    
    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>

    <div id="navbar" class="collapse navbar-collapse ms-3">
      <ul class="navbar-nav me-auto">
<li class="nav-item">
  <a class="nav-link" href="../index.html">
    <span class="fa fa-home fa-lg"></span>
     
  </a>
</li>
<li class="nav-item dropdown">
  <a href="#" class="nav-link dropdown-toggle" data-bs-toggle="dropdown" role="button" aria-expanded="false" aria-haspopup="true" id="dropdown-course-materials">Course Materials</a>
  <div class="dropdown-menu" aria-labelledby="dropdown-course-materials">
    <a class="dropdown-item" href="../articles/sheets/sci_articles/BIOL210_Syllabus_Fall2024_Roles.pdf">Syllabus PDF</a>
    <a class="external-link dropdown-item" href="https://drive.google.com/drive/folders/1JYUwk8A0mCn_FV1q5LrELnBwP6ofVQwF?usp=drive_link">Content Videos</a>
    <a class="dropdown-item" href="../articles/problemsets.html">Problem Sets</a>
    <a class="dropdown-item" href="../articles/quizzes.html">Quizzes</a>
    <a class="dropdown-item" href="../articles/exams.html">Exams</a>
    <a class="dropdown-item" href="../midtermReflection.html">Midterm Reflection</a>
    <a class="dropdown-item" href="../finalReflection.html">Final Reflection</a>
    <a class="external-link dropdown-item" href="https://drive.google.com/drive/folders/1yKEEn8zpCMV7nXUnngNCCFMkezXpUCv4?usp=drive_link">QUAIL session info</a>
    <a class="external-link dropdown-item" href="https://aroles.github.io/biol211/">BIOL 211 Lab</a>
    <a class="external-link dropdown-item" href="https://drive.google.com/drive/folders/1QmV9ipj560BxzW37AS_NAuRU9uN_waEt?usp=drive_link">Student Folders</a>
    <a class="external-link dropdown-item" href="https://docs.google.com/spreadsheets/d/1OHOpksRrvnDebY0Sy5ALFL6Pvb5j0NEFvAcbDdM77eM/edit?usp=drive_link">Check Progress Spreadsheet</a>
    <div class="dropdown-divider"></div>
  </div>
</li>
<li class="nav-item dropdown">
  <a href="#" class="nav-link dropdown-toggle" data-bs-toggle="dropdown" role="button" aria-expanded="false" aria-haspopup="true" id="dropdown-course-agenda">Course Agenda</a>
  <div class="dropdown-menu" aria-labelledby="dropdown-course-agenda">
    <a class="dropdown-item" href="../articles/agenda-week0.html">Week 0, Aug 30</a>
    <a class="dropdown-item" href="../articles/agenda-week1.html">Week 1, Sep 4-6</a>
    <a class="dropdown-item" href="../articles/agenda-week2.html">Week 2, Sep 9-13</a>
    <a class="dropdown-item" href="../articles/agenda-week3.html">Week 3, Sep 16-20</a>
    <a class="dropdown-item" href="../articles/agenda-week4.html">Week 4, Sep 23-27</a>
    <a class="dropdown-item" href="../articles/agenda-week5.html">Week 5, Sep 30-Oct 4</a>
    <a class="dropdown-item" href="../articles/agenda-week6.html">Week 6, Oct 7-11</a>
    <a class="dropdown-item" href="../articles/agenda-week7.html">Week 7, Oct 14-18</a>
    <div class="dropdown-divider"></div>
    <h6 class="dropdown-header" data-toc-skip>Week 8, Oct 21-25 - Fall Break</h6>
    <div class="dropdown-divider"></div>
    <a class="dropdown-item" href="../articles/agenda-week9.html">Week 9, Oct 28-Nov 1</a>
    <a class="dropdown-item" href="../articles/agenda-week10.html">Week 10, Nov 4-8</a>
    <a class="dropdown-item" href="../articles/agenda-week11.html">Week 11, Nov 11-15</a>
    <a class="dropdown-item" href="../articles/agenda-week12.html">Week 12, Nov 18-22</a>
    <a class="dropdown-item" href="../articles/agenda-week13.html">Week 13, Nov 25-27</a>
    <a class="dropdown-item" href="../articles/agenda-week14.html">Week 14, Dec 2-6</a>
    <a class="dropdown-item" href="../articles/agenda-week15.html">Week 15, Dec 9-11</a>
    <div class="dropdown-divider"></div>
  </div>
</li>
<li class="nav-item">
  <a class="external-link nav-link" href="https://docs.google.com/forms/d/e/1FAIpQLScJMEW72xvMW2KMuCenetBqUhi-AXqvbI5okDjlbV6hEdkIsQ/viewform?usp=sf_link">Assignment Turn In</a>
</li>
      </ul>
<form class="form-inline my-2 my-lg-0" role="search">
        <input type="search" class="form-control me-sm-2" aria-label="Toggle navigation" name="search-input" data-search-index="../search.json" id="search-input" placeholder="Search for" autocomplete="off">
</form>

      <ul class="navbar-nav">
<li class="nav-item">
  <a class="nav-link" href="../articles/sheets/sci_articles/BIOL210_Syllabus_Fall2024_Roles.pdf">BIOL 210 Fall 2024 Syllabus PDF</a>
</li>
      </ul>
</div>

    
  </div>
</nav><div class="container template-article">


<div class="row">
  <main id="main" class="col-md-9"><div class="page-header">
      <img src="" class="logo" alt=""><h1>BIOL 210 Problem Set 7 KEY</h1>
            
            <h4 data-toc-skip class="date">Week 7, Oct 14-18</h4>
      
      
      <div class="d-none name"><code>ps7key.Rmd</code></div>
    </div>

    
    
<p><text style="background-color:lightcyan">This is the key for PS 7.
Before reading this document, you should have completed the problems.
Use this key to check and correct your work BEFORE submitting the
corrected version via the google form.</text></p>
<p><text style="background-color:lightcyan">You should compare your
responses with this key and make any changes in another font color. Be
sure to explain why you got things wrong (showing that now you
understand) as well as providing corrected responses.</text></p>
<div class="section level2">
<h2 id="question-key">Question Key<a class="anchor" aria-label="anchor" href="#question-key"></a>
</h2>
<ol>
<li>
<p>In humans, the MN blood group locus consists of 2 alleles: the M
allele and the N allele. These alleles encode antigens expressed on the
surface of red blood cells; the two alleles are codominant so that a
heterozygote (MN) has both the M-antigen and the N-antigen. In a study
in Poland, 3100 people were genotyped with the following results:</p>
<p>1126 people were MM</p>
<p>1446 people were MN</p>
<p>528 people were NN</p>
Calculate each of the following:
</li>
<ol type="a">
<li>
the observed genotype frequencies
</li>
<br><li>
the allele frequencies
</li>
<br><li>
the expected number of individuals of each genotype (assuming random
mating)
</li>
<br><li>
carry out a Chi-square test for goodness of fit to random-mating
proportions (with 1 degree of freedom). Is this population in
Hardy-Weinberg Proportions for this locus? Explain.
</li>
</ol>
<p><text style="background-color:lightcyan">To get the observed genotype
frequencies, we just divide the number of individuals of that genotype
by the total number of individuals. Here’s the calcultion for
MM:</text></p>
<p><text style="background-color:lightcyan">obs freq MM = 1126 / 3100 =
0.36</text></p>
<p><text style="background-color:lightcyan">To get the expected
frequency of a genotype, we have to figure out the allele frequencies
and use those to calculate the HW expected fraction. To get allele
frequencies, we can use our observed genotype frequencies as
follows:</text></p>
<p><text style="background-color:lightcyan">For the M allele, 100% of
the alleles in MM individuals have the M allele and half of the alleles
in MN individuals are M alleles so we get:</text></p>
<p><text style="background-color:lightcyan">p = freq M = 0.36*1 +
0.47*0.5 = 0.6</text></p>
<p><text style="background-color:lightcyan">Then we can calculate the
frequency of the N allele by doing 1 - p OR we can do it the same way we
did with M:</text></p>
<p><text style="background-color:lightcyan">q = freq N = 0.17*1 +
0.47*0.5 = 0.4</text></p>
<p><text style="background-color:lightcyan">Now, we use our allele
frequencies to find out how many MM, MN, and NN individuals we’d expect,
given how many of each type of allele we have. We expect to get MM as
often as you expect to draw two M alleles by chance, or p x p =
p<sup>2</sup> = 0.6 x 0.6 = 0.36.</text></p>
<p><text style="background-color:lightcyan">In this example, our
expected frequency and our observed frequency turn out to be quite
similar! For MN you use 2pq and for MM you use q<sup>2</sup>.</text></p>
<p><text style="background-color:lightcyan">Once we have expected
frequencies, we multiply them by the total number of individuals sampled
to see how many individuals we’d expect to have found with each
genotype. For MM, 0.36 x 3100 = 1116 individuals expected.</text></p>
<p><text style="background-color:lightcyan">Our last calculation to do
here is to say how different are our observed and expected numbers of
individuals. This is where we do the calculation <span class="math inline">\(\frac{(O-E)^2}{E}\)</span>. We subtract the
expected number from the observed, square the value, and divide by the
expected number. Here’s the calculation for MM:</text></p>
<p><text style="background-color:lightcyan">(1126 -
1116)<sup>2</sup>/1116 = 10<sup>2</sup>/1116 = 0.089</text></p>
<p><text style="background-color:lightcyan">We perform the calculation
for each genotype and then sum them up, as in the table below, to get
our <span class="math inline">\(\chi^2\)</span> value of
3.339.</text></p>
<table class="table  lightable-paper" style='font-family: "Arial Narrow", arial, helvetica, sans-serif; margin-left: auto; margin-right: auto;'>
<thead><tr>
<th style="text-align:left;">
Genotype
</th>
<th style="text-align:right;">
ObsNo
</th>
<th style="text-align:right;">
ObsFreq
</th>
<th style="text-align:right;">
ExpFreq
</th>
<th style="text-align:right;">
ExpNo
</th>
<th style="text-align:right;">
(O-E)^2/E
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
MM
</td>
<td style="text-align:right;">
1126
</td>
<td style="text-align:right;color: blue !important;">
0.36
</td>
<td style="text-align:right;color: blue !important;">
0.36
</td>
<td style="text-align:right;color: blue !important;">
1116
</td>
<td style="text-align:right;color: blue !important;">
0.089
</td>
</tr>
<tr>
<td style="text-align:left;">
MN
</td>
<td style="text-align:right;">
1446
</td>
<td style="text-align:right;color: blue !important;">
0.47
</td>
<td style="text-align:right;color: blue !important;">
0.48
</td>
<td style="text-align:right;color: blue !important;">
1488
</td>
<td style="text-align:right;color: blue !important;">
1.185
</td>
</tr>
<tr>
<td style="text-align:left;">
NN
</td>
<td style="text-align:right;">
528
</td>
<td style="text-align:right;color: blue !important;">
0.17
</td>
<td style="text-align:right;color: blue !important;">
0.16
</td>
<td style="text-align:right;color: blue !important;">
496
</td>
<td style="text-align:right;color: blue !important;">
2.065
</td>
</tr>
<tr>
<td style="text-align:left;">
Total
</td>
<td style="text-align:right;">
3100
</td>
<td style="text-align:right;color: blue !important;">
1.00
</td>
<td style="text-align:right;color: blue !important;">
1.00
</td>
<td style="text-align:right;color: blue !important;">
3100
</td>
<td style="text-align:right;color: blue !important;">
3.339
</td>
</tr>
</tbody>
</table>
<p><text style="background-color:lightcyan">Finally, we interpret our
value. This population is in Hardy-Weinberg Proportions for this locus
since the estimated <span class="math inline">\(\chi^2\)</span> value is
less than the critical value of 3.84 for 1 df. Thus, the difference
between observed and expected values is due only to chance with P &gt;
0.05, meaning that there is more than a 5% chance that the difference
between observed and expected is due to chance. Note that it’s important
to provide not just the estimated test statistic but also the P-value as
well as your interpretation of what this result means.</text></p>
<br><li>
Consider the data presented by Jorde and Wooding (2004; only 6 pages
long). Do human populations differ genetically? How do we know?
</li>
<p><text style="background-color:lightcyan">Jorde and Wooding provide a
variety of ways to understand or measure genetic variation in the human
population. For example, they provide the estimate that a randomly
chosen pair of humans will differ from each other (on average) for about
1 in every 1000-1500 base pairs of DNA (this amounts to 2-3 million base
pairs in the whole genome). This is low compared to many species (see
the content video on genetic variation).</text></p>
<p><text style="background-color:lightcyan">The numbers just provided
are about genetic variation in the whole human population but this
question is asking whether DIFFERENT human populations are also
genetically different from each other (measurably). So, if we consider
the base pairs (or nucleotides) in the genome where humans vary one from
another, what do we see? We find that there is some geographic signal in
the data – people from different locations in the world do tend to
differ for some places in the genome. In fact, about 10-15% of the
variation in the whole species is correlated with geographic location
(continental groups of Africa, Asia, Europe). Meanwhile, 85-90% of the
variation within humanity is common to most human populations, it is
shared by populations in different locations. As a result, we see that
there is very little genetic difference between different peoples and
much much more of our variation is found in all populations.</text></p>
<p><text style="background-color:lightcyan">These data result from
sequencing different parts of the human genome and comparing them for
human genome samples from all over the world. Jorde and Wooding describe
3 kinds of genetic loci (not genes but locations in the genome, may be
non-coding DNA): STR polymorphisms (Short Tandem Repeats), restriction
site polymorphisms, and Alu insertion polymorphisms. All of these loci
represent sequences that do not code for proteins and thus will evolve
neutrally, not likely to have a functional effect on the organism’s
phenotype. </text><br></p>
<li>
Refer to both Jorde and Wooding (2004) and/or Sankar and Cho (2002; only
2 pages long) to answer the following questions:
</li>
<ol type="a">
<li>
How might we define “race”? What does this term represent? Is it useful?
Harmful? Does it have any biological meaning? How might we decide
whether there’s biological relevance or what role biology might play
here? (I’m not looking for some right answer. I would like you to
reflect on this in light of the readings and your experiences—and also
as potential biologists!)
</li>
<p><text style="background-color:lightcyan">Responses will vary. Jorde
and Wooding seem to largely argue that “race” is pretty much entirely a
social construct that does more harm than good. They show evidence that
the way that “race” is defined has changed enormously over time and
given the contexts in which it is used is usually causing harm, defining
differences that are then used predjudically. Given the actual data on
human genetic variation, J&amp;W seem to argue that race is not
biologically based or relevant.</text><br></p>
<li>
What might be benefits or costs of using a term like “race” (or racial
descriptors) when discussing human genetic variation? If not by using a
term like race, do you think there are better ways we could describe
groups of humans that differ genetically from each other?
</li>
<text style="background-color:lightcyan">Responses will vary. As noted
above, J&amp;W give evidence (and you may already be aware) of the ways
in which race has been used to cause harm historically as well as today.
Since race has been used to divide and justify power differentials, it
has great potential for harm. Those who argue for a benefit to using
race as a descriptor may argue that there are health benefits, say to
being aware that a given race has a higher or lower risk of some health
conditions compared to other races. However, higher or lower risk
doesn’t mean a person must get a condition or absolutely can’t have it.
Nor does it mean that a medication absolutely will or will not work on a
person of a given race. As noted above, there is TONS more variation
between people overall compared to the number of differences between
folks from different parts of the world. This reflects the fact that the
characeristics that define race (often primarily skin color) are related
to very few genes in the genome. Thus, it’s not logical to assume that
you know the rest of someone’s genome (and therefore health risks) based
just on their skin color. Given everything we’ve talked about all
semester, I hope you can appreciate that the environment has an equally
large role in affecting phenotype, especially for things like health. In
terms of other ways to describe groups of humans, J&amp;W suggest that
the best way to do something like determine health risks is to examine
the individual’s genotype information as well as their lifestyle factors
that are known to impact a given health condition.</text>
</ol>
</ol>
<p><br></p>
<p><b>When you are finished, check your responses on the <a href="ps7key.html">key for PS7</a>.</b></p>
<p>Remember to sign the Honor Code on your assignment.</p>
</div>
  </main><aside class="col-md-3"><nav id="toc"><h2>On this page</h2>
    </nav></aside>
</div>



    <footer><div class="pkgdown-footer-left">
  <p>Developed by Prof. A. Roles.</p>
</div>

<div class="pkgdown-footer-right">
  <p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.9.</p>
</div>

    </footer>
</div>

  

  

  </body>
</html>
